docs/source/science/NISAR/Simulated_NISAR.ipynb

```diff
@@ -0,0 +1,11823 @@
+{
```

Comment:
<p>Consider specifying <em>which</em> year, perhaps in parentheses, so that future readers know which year, until this notebook is perhaps updated after the launch actually occurs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>